backend/hct_mis_api/apps/grievance/tests/snapshots/snap_test_filter_grievance_by_cross_area.py

```diff
@@ -77,16 +77,6 @@
                         'language': 'Polish',
                         'status': 1
                     }
-                },
-                {
-                    'node': {
-                        'admin': 'Admin Area 2',
-                        'category': 8,
-                        'consent': True,
-                        'description': 'Same Area Grievance',
-                        'language': 'Polish',
-                        'status': 1
-                    }
```

Comment:
This test is skipped, so please do not update this because this is incorrect result.